bridge/github/config.go

```diff
@@ -217,36 +387,42 @@ func promptURL() (string, string, error) {
 		}
 
 		line = strings.TrimRight(line, "\n")
-
 		if line == "" {
 			fmt.Println("URL is empty")
 			continue
 		}
 
-		projectUser, projectName, err := splitURL(line)
-
+		// get owner and project from url
+		_, owner, project, err := splitURL(line)
 		if err != nil {
 			fmt.Println(err)
 			continue
 		}
 
-		return projectUser, projectName, nil
+		return owner, project, nil
 	}
 }
 
-func splitURL(url string) (string, string, error) {
-	re, err := regexp.Compile(`github\.com\/([^\/]*)\/([^\/]*)`)
-	if err != nil {
-		panic(err)
+func splitURL(url string) (string, string, string, error) {
+	res := rxGithubURL.FindStringSubmatch(url)
+	if res == nil {
+		return "", "", "", fmt.Errorf("bad github project url")
 	}
 
-	res := re.FindStringSubmatch(url)
+	return res[0], res[1], res[2], nil
+}
 
-	if res == nil {
-		return "", "", fmt.Errorf("bad github project url")
+func valideGithubURLRemotes(remotes map[string]string) []string {
```

Comment:
a `validateXXX` function would return a bool or an error, but you are returning an array. A better name is needed!

Reply:
Changed the name to `getValidateGithubRemoteURLs`